src/components/Square.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
+  // For Wave 1 enable this
+  //  Component to alert a parent
   //  component when it's clicked on.
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return (
+    <button className="square" onClick={() => props.onClickCallback(props.id)}>
```

Comment:
Nice inline handler here. Notice that we can't simply use the props.onClickCallback as the handler, since we need to pass it the id of the square that was clicked, while any event handler will be passed the event details from the browser instead!